MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/styles/TransparentBackgroundActivity.java

```diff
@@ -0,0 +1,135 @@
+
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.RawRes;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.VideoView;
+
+import com.mapbox.mapboxandroiddemo.R;
+import com.mapbox.mapboxsdk.Mapbox;
+import com.mapbox.mapboxsdk.maps.MapView;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.io.StringWriter;
+import java.io.Writer;
+
+import timber.log.Timber;
+
+/**
+ * Create a transparent render surface and add whatever you want. This example has a moving water.
+ */
+public class TransparentBackgroundActivity extends AppCompatActivity implements OnMapReadyCallback {
+
+  private MapView mapView;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+
+    // Mapbox access token is configured here. This needs to be called either in your application
+    // object or in the same activity which contains the mapview.
+    Mapbox.getInstance(this, getString(R.string.access_token));
+
+    // This contains the MapView in XML and needs to be called after the access token is configured.
+    setContentView(R.layout.activity_style_transparent_render_background);
+
+    mapView = findViewById(R.id.mapView);
+    mapView.onCreate(savedInstanceState);
+    mapView.getMapAsync(this);
+  }
+
+  @Override
+  public void onMapReady(MapboxMap mapboxMap) {
+    try {
+      // Switch the map to a style that has no background
+      mapboxMap.setStyleJson(readRawResource(this, R.raw.no_bg_style));
+      initVideoView();
+    } catch (IOException exception) {
+      Timber.e(exception);
+    }
+  }
+
+  /**
+   * Place the video of moving water behind the map
+   */
+  private void initVideoView() {
+    VideoView backgroundWaterVideoView = findViewById(R.id.videoView);
+    String path = "android.resource://" + getPackageName() + "/" + R.raw.moving_background_water;
+    backgroundWaterVideoView.setVideoURI(Uri.parse(path));
+    backgroundWaterVideoView.start();
+    backgroundWaterVideoView.setOnCompletionListener(mediaPlayer ->
+      backgroundWaterVideoView.start()
+    );
+  }
+
+  // Add the mapView lifecycle to the activity's lifecycle methods
+  @Override
+  public void onResume() {
+    super.onResume();
+    mapView.onResume();
+  }
+
+  @Override
+  protected void onStart() {
+    super.onStart();
+    mapView.onStart();
+  }
+
+  @Override
+  protected void onStop() {
+    super.onStop();
+    mapView.onStop();
+  }
+
+  @Override
+  public void onPause() {
+    super.onPause();
+    mapView.onPause();
+  }
+
+  @Override
+  public void onLowMemory() {
+    super.onLowMemory();
+    mapView.onLowMemory();
+  }
+
+  @Override
+  protected void onDestroy() {
```

Comment:
Given that you start the video onCreate 
Should you stop it in onDestroy? 
I would be curious to see how it works on a tablet on Orientation change. My tablet is at home in PA though :(

Reply:
Yea, I should stop it in `onDestroy()`. Fix made and pushed to this pr.

Still funky when in tablet landscape though 😞 


![ezgif com-optimize 6](https://user-images.githubusercontent.com/4394910/44118956-8c087e50-9fcc-11e8-828d-59216e36ea3f.gif)

Reply:
@langsmith What happens if you start in landscape? Is it ok then?

Reply:
Made your suggestions regarding `onDestroy()`

Creation/destruction seems to be 👌 

![ezgif com-resize](https://user-images.githubusercontent.com/4394910/44372592-83069b80-a499-11e8-9b85-eb37f7e9e755.gif)